core/src/main/java/de/jplag/strategy/AbstractComparisonStrategy.java

```diff
@@ -46,7 +41,6 @@ protected void compareSubmissionsToBaseCode(SubmissionSet submissionSet) {
      */
     protected Optional<JPlagComparison> compareSubmissions(Submission first, Submission second) {
         JPlagComparison comparison = greedyStringTiling.compare(first, second);
-        logger.info("Comparing {}-{}: {}", first.getName(), second.getName(), comparison.similarity());
```

Comment:
Could removing this result in having a long time without a log in large submission sets? If so I think we should still keep it and maybe just remove the similarity or add that it is only a temporary value.
Otherwise outside users might have the feeling that JPlag is not working.